tasks/users.yml

```diff
@@ -8,7 +8,7 @@
 - name: PostgreSQL | Make sure the PostgreSQL users are present
   postgresql_user:
     name: "{{item.name}}"
-    password: "{{ item.pass | default(omit) }}"
```

Comment:
Is this backwards compatible with existing inventories?  We should be weary about introducing breaking changes.

Is there a simple way to say:

if `item.pass` is defined, then use `item.pass`
else if `item.password` is defined, then use `item.password`
else omit